kwasm-lemmas.md

```diff
@@ -390,21 +434,34 @@ module WRC20-LEMMAS
     imports KWASM-LEMMAS
 ```
 
-This conversion turns out to be helpful in this particular proof, but we don't want to apply it on all KWasm proofs.
+These conversions turns out to be helpful in this particular proof, but we don't want to apply it on all KWasm proofs.
 
 ```k
-    rule X /Int 256 => X >>Int 8
+    rule X /Int N => X >>Int 8  requires N ==Int 256 [simplification]
 ```
 
 TODO: The two `#get` theorems below theorems handle special cases in this proof, but we should be able to use some more general theorems to prove them.
 
 ```k
-    rule (#get(BM, ADDR) +Int (X +Int Y)) modInt 256 => (#get(BM, ADDR) +Int ((X +Int Y) modInt 256)) modInt 256       [simplification]
-    rule (#get(BM, ADDR) +Int X)           >>Int 8   => X >>Int 8 requires X modInt 256 ==Int 0 andBool #isByteMap(BM) [simplification]
+    rule (#get(BM, ADDR) +Int (X +Int Y)) modInt N => (#get(BM, ADDR) +Int ((X +Int Y) modInt 256)) modInt 256
+      requires N ==Int 256
+      [simplification]
```

Comment:
Same, `256` can go directly in the rule.

Reply:
Should this have been resolved?